docs/available_commands.md

```diff
@@ -80,6 +80,10 @@ The *target component* can be identified through any of the identifiable propert
 
 If *coordinates* are used to identify the target, they must match the component fully. In other words, if __only__ *name* is given, it will __only match__ components with that name which do __not__ contain *version* or *group* fields.
 
+In *coordinates* it is also possible to provide a range of versions following the [PURl specification](https://github.com/package-url/purl-spec/blob/version-range-spec/VERSION-RANGE-SPEC.rst) as referenced by [CycloneDX](https://cyclonedx.org/docs/1.6/json/#vulnerabilities_items_affects_items_versions_items_range). Further Information on the supported versioning schemas can be found here [univers documentation](https://pypi.org/project/univers/).
+
+It is also possible to use a wildcard by setting the *version* as "*", in that case the version will be ignored and only *name*and *group* will be considered.
```

Comment:
Same comment as in `identity.py`. Why this special case? The *vers* specification already allows for the `*` syntax.

Reply:
Because the vers specification allows * only in accordance with the specified versioning schema. 
Since some people use their own versioning schema it was a consideration for this.

Now changed taking recent discussions and 'safety' considerations into account. 
Since univers is open source, people can try to add their schema or switch to a more common one.